core/domain/role_services.py

```diff
@@ -131,3 +142,71 @@ def get_all_actions(role):
         role_actions.extend(get_all_actions(parent_role))
 
     return list(set(role_actions))
+
+
+def get_updatable_roles():
+    """Returns roles to which a user's role can be updated by admin interface.
+
+    Returns:
+        dict. A dict of all roles that a user can be given by admin.
+            key -> roles in format as stored in backed.
+            value -> roles in format to show users.
+    """
+    updatable_roles = {}
+    for role in PARENT_ROLES:
+        if role not in [feconf.ROLE_GUEST]:
```

Comment:
I think I did that as I wasn't sure if it should be [feconf.ROLE_GUEST] or [feconf.ROLE_GUEST, feconf.ROLE_SUPER_USER]?
Mainly, are we going to allow super users other than site_admin (super_user would access the admin's page).
@seanlip .. your thoughts on this?